### PR TITLE
Devex rebuild cognito

### PR DESCRIPTION
**Pull request by zachrog**
We can no longer destroy and recreate our Cognito due to an incorrect configuration.

Dropping OAuth(which is unused) allows us to rebuild cognito

### CODE REVIEW

**Review by En-8 — APPROVED** [review]

**Review by JayFlexy — APPROVED** [review]

**Comment by jimlerza** [comment]
Added the "Do Not Merge" label because we need to verify with the IRS that this doesn't break eTCS before we merge this.

**Review by TomElliottFlexion — APPROVED** [review]

**Pull request merged**
